src/workloads/query/Lookup.yml

```diff
@@ -326,3 +326,8 @@ AutoRun:
         $eq:
           - shard
           - shard-all-feature-flags
+          - 1-shard-dedicated.2024-05
```

Comment:
How do the changes in the workload files relate to the task changes above? I can't tell how these are supposed to correlate. Does this list mean: schedule this task whenever we're running one of these configs? And if so, then what do the changes in `genny_tasks.yml` mean?

Reply:
I think this is related to some history with how Genny tasks were historically scheduled, where the auto* task may run on variants where some of the individual workloads were not compatible. But yes at a high level I believe this must match the `mongodb_setup` for each of the Genny tasks where this workload is included. [Here is a patch](https://spruce.mongodb.com/task/genny_validate_genny_tasks_validate_genny_tasks_patch_46408e2d9fabae647be29746fe3e04560e03a286_674df8801a9b7d0007e5ecec_24_12_02_18_12_17/logs?execution=0) where they don't align and the error `To resolve this issue, please run ./run-genny auto-tasks-local and commit the new files in the evergreen folder.`


What's nice is that you can add the conditions here and then run `./run-genny auto-tasks-local` to automatically populate the `genny_tasks.yml`. Hopefully that makes sense but I'm certainly not the expert on this!